Dockerfile

```diff
@@ -37,16 +39,17 @@ RUN apk add --no-cache \
     py3-crcmod \
     rsync \
     util-linux \
-    zip 
+    zip
 
 RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
 ENV NVM_DIR /root/.nvm
 ENV PATH $NVM_DIR:$PATH
 RUN chmod +x $NVM_DIR/nvm.sh
 RUN $NVM_DIR/nvm.sh --version
-RUN $NVM_DIR/nvm.sh install 8
-RUN $NVM_DIR/nvm.sh install 10
-RUN $NVM_DIR/nvm.sh install 12
+RUN $NVM_DIR/nvm.sh install 18
```

Comment:
Is this the installed node versions? Would it cause issues to anything using node16 for example?

Reply:
Node 16 is deprecated by google now anyway, but its only preinstalled, anyone can install it if there's a one-off need.

Reply:
<img width="370" alt="Screenshot 2024-04-24 at 16 01 36" src="https://github.com/PlaySportsGroup/docker-buildtools/assets/630539/a394e2cc-b19b-47d2-b5b1-f40906542f1d">